networking/src/NTP/Client.hs

```diff
@@ -39,16 +38,19 @@ import           System.Wlog (LoggerName, WithLogger, logDebug, logError, logInf
                               modifyLoggerName)
 
 import           Mockable.Class (Mockable)
-import           Mockable.Concurrent (Async, Concurrently, Delay, concurrently, forConcurrently,
+import           Mockable (Async, Concurrently, CurrentTime, Delay, concurrently, currentTime, forConcurrently,
                                       timeout, withAsync)
 import           NTP.Packet (NtpPacket (..), evalClockOffset, mkCliNtpPacket, ntpPacketSize)
 import           NTP.Util (createAndBindSock, resolveNtpHost, selectIPv4, selectIPv6,
                            udpLocalAddresses, withSocketsDoLifted)
 
-data NtpClientSettings m = NtpClientSettings
+data NtpStatus = NtpSyncOk | NtpDesync Microsecond
```

Comment:
This is I guess where we'd want to add `NtpSyncUnavailable`, right?

Reply:
Yes indeed, but we might not need it after all.  I think you agreed that we only need to disable/bypass the startup ntp check rather than disable wallet server to query for ntp status, hence there is no need for additional status message.

Reply:
I can't remember. I thought the idea was to either disable it completely (in which case UI client should get a `Unavailable` variant) or to enable it (in which case checks run in the background and fill the status var).

Reply:
As I just explained in slack (so just for completeness sake), there is no background ntp server running, every time wallet asks for ntp status, the backend spawns a new ntp client.  Thus there is not need to have `NtpSyncUnavailable`.

But I am open for changing the ntp architecture, just not in this PR :)